src/toil_rnaseq_sc/rnaseq_sc_cgl_pipeline.py

```diff
@@ -58,14 +60,14 @@ def parse_samples(path_to_manifest):
                 [require(urlparse(x).scheme in SCHEMES,
                          'URL "{}" not valid. Schemes:{}'.format(url, SCHEMES)) for x in url]
             else:
-                raise UserError('URL is ')
+                raise UserError('URL is inot in proper form: .tar.gz, .tar, .fastq.gz, .fastq, .fq.gz, .fq')
```

Comment:
Also — the fastq formats I listed above don't include the `fastq.tst.gz` format, which is problematic.